docs/virtual_machines/disks_and_volumes.md

```diff
@@ -412,6 +412,56 @@ spec:
         claimName: mypvc
 ```
 
+
+#### block device from host
+
+to use a local block device from a host, you need to either have a provisioner that supports block devices such as openebs localpv,
+or create a pv by hand like so:
+
+```yaml
+---
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: local-device
+provisioner: kubernetes.io/no-provisioner
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: sdb
+spec:
+  volumeMode: Block
+  storageClassName: local-device
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 100Gi
+---
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: sdb
+spec:
+  volumeMode: Block
+  storageClassName: local-device
+  nodeAffinity:
+    required:
+      nodeSelectorTerms:
+      - matchExpressions:
+        - key: kubernetes.io/hostname
+          operator: In
+          values:
+          - my-node
+  accessModes:
+    - ReadWriteOnce
+  capacity:
+    storage: 100Gi
+  local:
+    path: /dev/sdb
```

Comment:
Please, split the storage class and PV creation to the PVCs definition. We should make clear here that PVs ans SCs can only be created by the cluster-admin and not by regular users. Regular users can only get storage classes and create PVCs